src/components/UserRewardDisplay.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from "react";
+import { IUserReward } from "../models/User";
+import './App.css';
+import '@fontsource/roboto/300.css';
```

Comment:
Remove imports on lines 3-7. These are already included in the app.

Reply:
Got it.